src/grt/src/fastroute/src/maze.cpp

```diff
@@ -69,11 +68,10 @@ void FastRouteCore::convertToMazerouteNet(int netID)
     y1 = treenodes[n1].y;
     x2 = treenodes[n2].x;
     y2 = treenodes[n2].y;
-    treeedge->route.gridsX = (short*) calloc((edgelength + 1), sizeof(short));
-    treeedge->route.gridsY = (short*) calloc((edgelength + 1), sizeof(short));
-    treeedge->route.gridsL = nullptr;
-    gridsX = treeedge->route.gridsX;
-    gridsY = treeedge->route.gridsY;
+    treeedge->route.gridsX.resize(edgelength + 1);
+    treeedge->route.gridsY.resize(edgelength + 1);
```

Comment:
I've updated the `resize` call to fill the allocated memory space with zeros. Since most of the `resize` calls run after a `clear`, I think the behavior will be the same of calloc.